### PR TITLE
Oop tests 2

### PR DESCRIPTION
**Pull request by stephanieharrison-maximus**
### Please check if the PR fulfills these requirements:

- [x] After these changes, the app was run and still works as expected
- [x] Tests for these changes were added (if applicable)
- [x] All existing unit tests were run and still pass
- [x] Workflow runs without error

### Please specify the type of change this PR introduces (Bug fix, feature addition, content update, chore, etc.):

Add more unit tests for OOP. Also install Eslint for Jest and update API endpoints so dummy data works properly.

### Additional Notes:

Three tests were skipped because they don't work yet, but if someone in the future comes back to making tests for OOP, they may find the unfinished work useful for that. If this is a problem, I can remove the unfinished tests, though.

### CODE REVIEW

File: oop/vue.config.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ module.exports = {
   devServer: {
     proxy: {
       '/api': {
-        target: 'https://oop-web-a3c641-dev.apps.silver.devops.gov.bc.ca',
+        target: 'https://oop-web-a3c641-test.apps.silver.devops.gov.bc.ca',
```
Comment (harrymaynard-maximus) [inline oop/vue.config.js:6]: Please change this back to "dev". We still want to use the DEV environment for local development.

**Review by harrymaynard-maximus — CHANGES_REQUESTED** [review]
One observation: There's a bunch of formatting changes in this PR, so it's difficult to determine what I've already reviewed and what's new. In the future, please try to reduce the amount of formatting changes in existing code. It takes time to go through 3900+ changes.

File: oop/vue.config.js (hunk 1 above)
Comment (stephanieharrison-maximus) [reply oop/vue.config.js]: Updated and pushed

**Review by harrymaynard-maximus — APPROVED** [review]
Looks good. Approved.

**Pull request merged**